tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
@everilae: the asymmetry still doesn't make sense to me. It's unintuitive that I may pass a string for q_name, but it would come back as bytes in the result and that the two would not compare equal in py3.x. I understand that bytes and str are not equivalent in 3.x, but this asymmetry in pika seems rather error-prone to me. For the life of me, I can't seem to find the definitions of basic AMQP types such as `shortstr`. Would you mind pointing me to where they are defined? Is `shortstr` defined as UTF8 or just binary bytes in AMQP?